### PR TITLE
netty: bump to 4.1.32 and tcnative 2.0.20

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
This is a revival of @ST-DDT 's issue #5080

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -223,7 +223,7 @@ subprojects {
             // Keep the following references of tcnative version in sync whenever it's updated
             // SECURITY.md (multiple occurrences)
             // examples/build.gradle
-            netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:2.0.17.Final',
+            netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:2.0.20.Final',
```
Comment (dapengzhang0) [inline build.gradle]: Sync these two: `SECURITY.md (multiple occurrences)` and `examples/build.gradle`

Comment (carl-mastrangelo) [reply build.gradle]: Done.

Comment (dapengzhang0) [reply build.gradle]: LGTM

**Comment by dapengzhang0** [comment]
There are multiple occurrences in `repositories.bzl` as well.

**Pull request merged**